shaders/outlinepattern.fragment.glsl

```diff
@@ -21,5 +21,13 @@ void main() {
     vec2 pos2 = mix(u_pattern_tl_b, u_pattern_br_b, imagecoord_b);
     vec4 color2 = texture2D(u_image, pos2);
 
-    gl_FragColor = mix(color1, color2, u_mix) * u_opacity;
+    // find distance to outline for alpha interpolation
+    float dist_a = length(v_pos_a - gl_FragCoord.xy);
```

Comment:
In order to fade the pixel appropriately we need the distance of the pixel from the center of the line. `gl_FragCoord.xy` is the position of the pixel which we need to subtract from the position of the closest point on the line.

`v_pos_a` is the position of texture pixel we need. I think we need another `varying v_pos` that has the position of the nearest point on the line like in the outline shader.